### PR TITLE
Feat/improve themes

### PR DESCRIPTION
**Pull request by rohitdasu**

### CODE REVIEW

**Comment by rupali-codes** [comment]
@rohitdasu can we change side navbar element border color for light theme?

File: components/TopBar/TopBar.tsx [hunk 1]
```diff
@@ -1,11 +1,39 @@
...
+  const category = router.asPath.replace("/", "");
+  return (
+    <div className="flex items-center justify-between mb-4">
+      <div className="flex items-center text-3xl my-4 dark:text-gray-300">
+        <FaSlackHash className="mt-1 mr-2 text-gray-600 dark:text-gray-300" />
+        <span className="text-gray-900 dark:text-gray-100 uppercase font-bold">
+          {header ?? category}
+        </span>
+      </div>
+      <div className="items-center flex justify-center gap-2">
```
Comment (rupali-codes) [inline components/TopBar/TopBar.tsx:17]: reduce this gap a bit, 
image

**Comment by rohitdasu** [comment]
@rupali-codes  updated the PR with requested changes.

**Comment by rupali-codes** [comment]
Thank you for the changes @rohitdasu

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**